kratos/tests/sources/test_nodal_data.cpp

```diff
@@ -80,12 +80,10 @@ namespace Kratos {
 					velocity_y_sum += i_node->FastGetSolutionStepValue(VELOCITY_Y);
 				}
 				KRATOS_CHECK_DOUBLE_EQUAL(pressure_sum, i * size);
-				KRATOS_CHECK_DOUBLE_EQUAL(velocity_x_sum, i * size);
+				KRATOS_CHECK_DOUBLE_EQUAL(velocity_x_sum, 2*i * size);
 				KRATOS_CHECK_DOUBLE_EQUAL(velocity_y_sum, 0.00);
 			}
 		}
```

Comment:
can you add a check that the key of (for example)

DISPLACEMENT_X, DISPLACEMENT_Y, DISPLACEMENT_Z is consecutive?